openslides_backend/action/actions/user/create.py

```diff
@@ -51,7 +51,7 @@ class UserCreate(
             "is_present_in_meeting_ids",
             "committee_management_ids",
             "is_demo_user",
-            "forwarding_committee_ids",
+            # "forwarding_committee_ids",
```

Comment:
missing documentation change of action and if you really want it, remove the line

Reply:
Can't change the documentation, because this field was never mentioned there in the first place.